docroot/themes/humsci/humsci_basic/src/scss/_main.scss

```diff
@@ -164,6 +164,7 @@
   'components/unpublished',
   'components/media',
   'components/private-page',
+  'components/views-layout',
```

Comment:
I think this work may be better suited placed under `objects` with other layout type styles. We have `_layouts.rows.scss`, let's make a `_layouts.views.scss` file for this.

![move placement](https://user-images.githubusercontent.com/12678977/108756640-c6cd3680-7516-11eb-8291-1f5be7706202.png)